bin/rigger-save-vars

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+export VARIABLES
+
+function usage {
+  cat <<EOF
+${BASH_SOURCE[0]} <variable name>
```

Comment:
Mostly depends if you intent on being `bash` only or letting people with `zsh` have some fun (heretics :P)